docs/community/com-query/grafana-connector.md

```diff
@@ -51,26 +58,26 @@ From `grafana-7-x.x/` run the following command:
 ### Add Datasource
 
 - Sign in or create credentials (you can use admin/admin if logging in for the first time)
-- Add Molecula as a datasource in Grafana and enter FeatureBase server information
+- Add Featurebase as a datasource in Grafana and enter FeatureBase server information
     - The welcome page should include a link to the [Add data source page](http://localhost:3000/datasources).
       - See the [Grafana docs](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source) for more details.
-    - If your molecula-datasource plugin is installed, it should appear in the data source list.
-
+    - If your featurebase-datasource plugin is installed, it should appear in the data source list.
 
 ![Grafana Welcome](/assets/images/grafana-welcome.png "Grafana Welcome")  
 *Click "Add your first data source"*
 
-![Grafana Add Datasource](/assets/images/grafana-add-datasource.png "Grafana Add Datasource")  
-*Select the "Molecula" option. The "unsigned" label is expected*
+![Grafana Add Datasource](/assets/images/grafana/add-datasource.png "Grafana Add Datasource")  
+*Select the "Featurebase" option. The "unsigned" label is expected*
 
 ### Configure Server
 
 Point Grafana to a running FeatureBase server
 - Default FeatureBase address: `localhost`
 - Default FeatureBase GRPC port: `20101`
+- Default FeatureBase HTTP port: `10101`
 - `Max Query Results` limits the number of records returned by the server if no `Limit` call is provided.
 
-![Grafana Configure Datasource](/assets/images/grafana-configure-server.png "Grafana Configure Datasource")  
+![Grafana Configure Datasource](/assets/images/grafana/configure-server.png "Grafana Configure Datasource")  
 *If Featurebase server is running, you will see a similar message indicating success*
```

Comment:
Another inherited issue.

Important information gets a "callout block" which is discussed here:

https://github.com/FeatureBaseDB/featurebase-docs/blob/main/help-on-help/writing-help/writing-content-callouts.md